packages/base/any/kernels/5.15-lts/patches/0001-feat-Add-dev-mvsw_fw_comm_debug-device-for-FW-debugg.patch

```diff
@@ -0,0 +1,680 @@
+commit 586d6dd722b331bc0500cbe0e239aef4110f8ca3
```

Comment:
Please create the patch file with `git format-patch` and do not use the output of `git show`.

Reply:
Updated the patch